opensrp-reveal/src/test/java/org/smartregister/reveal/shadow/SymbolLayerShadow.java

```diff
@@ -0,0 +1,19 @@
+package org.smartregister.reveal.shadow;
+
+import com.mapbox.mapboxsdk.style.layers.SymbolLayer;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+/**
+ * Created by samuelgithengi on 2/4/20.
+ */
+@Implements(SymbolLayer.class)
+public class SymbolLayerShadow extends LayerShadow {
+
+    @Implementation
+    public void __constructor__(String layerId, String sourceId) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name '__constructor__' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4935520)